settings.py

```diff
@@ -1134,6 +1139,14 @@
             'maxBytes': 10 * 1024 * 1024,  # 10 MB
             'backupCount': 200  # Backup 2000 MB of logs
         },
+        'kafka_audit': {
+            "level": "DEBUG" if KAFKA_PRODUCER_AUDIT_LOG_ENABLED else 'ERROR',
```

Comment:
Could this have been done by using `LOCAL_LOGGING_HANDLERS` without introducing a different localsetting?

Reply:
yes, that could be an alternative approach. I'll look into that.